templates/create_organisation.html

```diff
@@ -1,4 +1,20 @@
 {% extends "base.html" %}
+
+{% block extra_title %} Add Organisation{% endblock %}
+{% block extra_keywords %}, organisation, organisations, add, create{% endblock %}
+
+{% block metas %}
+<link rel="canonical" href="/add_business">
+<meta name="description" content="Add information for the  Dún Laoghaire-Rathdown County Council to hold about a new business.">
+<meta property="og:description" content="Add information for the  Dún Laoghaire-Rathdown County Council to hold about a new business." />
+<meta name="image" property="og:image" content="LINK TO IMAGE" />
+<meta property="og:image:alt" content="A form to add information to be held by Dún Laoghaire-Rathdown County Council about this business" />
+<meta property="og:url" content="/add_business" />
+<meta name="twitter:description" content="Add information for the  Dún Laoghaire-Rathdown County Council to hold about a new business.">
+<meta name="twitter:image" content="LINK TO IMAGE">
```

Comment:
Do we want to create a standalone issue for this to ensure we don't miss it later on in the development process and forget to update the content value @Abibubble ?

Reply:
My thought was to not close this issue with this PR. The issue has checkboxes for things that are done, which includes the things still left do to (like the images). Happy to split it up into multiple issues if that's preferred, not sure what best practice is on this (I'd normally do multiple issues, but didn't want to clutter the board up) :)